File: python/exercicios/ex004_tabuada.py
# Exercício 004 - Tabuada
# Objetivo: Praticar loops com for e range

numero = int(input("Digite um número para ver a tabuada: "))

print(f"\nTabuada do {numero}:")
for i in range(1, 11):
    print(f"{numero} x {i} = {numero * i}")

File: javascript/exercicios/ex004_tabuada.js
// Exercício 004 - Tabuada
// Objetivo: Praticar loops

function tabuada(numero) {
    console.log(`\nTabuada do ${numero}:`);
    for (let i = 1; i <= 10; i++) {
        console.log(`${numero} x ${i} = ${numero * i}`);
    }
}

tabuada(7);
